packages/gcc7.rb

```diff
@@ -36,125 +24,91 @@ class Gcc7 < Package
-      system "ln -sv #{CREW_PREFIX}/bin/gcc-nm-7.3 #{CREW_DEST_PREFIX}/bin/gcc-nm"
-      system "ln -sv #{CREW_PREFIX}/bin/gcc-ranlib-7.3 #{CREW_DEST_PREFIX}/bin/gcc-ranlib"
-      system "ln -sv #{CREW_PREFIX}/bin/gcov-7.3 #{CREW_DEST_PREFIX}/bin/gcov"
-      system "ln -sv #{CREW_PREFIX}/bin/gcov-dump-7.3 #{CREW_DEST_PREFIX}/bin/gcov-dump"
-      system "ln -sv #{CREW_PREFIX}/bin/gcov-tool-7.3 #{CREW_DEST_PREFIX}/bin/gcov-tool"
-      system "ln -sv #{CREW_PREFIX}/bin/gfortran-7.3 #{CREW_DEST_PREFIX}/bin/gfortran"
+      FileUtils.ln_s('gcc', "#{CREW_DEST_PREFIX}/bin/cc")
+      FileUtils.ln_s('gcc-7.4', "#{CREW_DEST_PREFIX}/bin/gcc")
+      FileUtils.ln_s('c++-7.4', "#{CREW_DEST_PREFIX}/bin/c++")
+      FileUtils.ln_s('g++-7.4', "#{CREW_DEST_PREFIX}/bin/g++")
+      FileUtils.ln_s('cpp-7.4', "#{CREW_DEST_PREFIX}/bin/cpp")
+      FileUtils.ln_s('gcc-ar-7.4', "#{CREW_DEST_PREFIX}/bin/gcc-ar")
+      FileUtils.ln_s('gcc-nm-7.4', "#{CREW_DEST_PREFIX}/bin/gcc-nm")
+      FileUtils.ln_s('gcc-ranlib-7.4', "#{CREW_DEST_PREFIX}/bin/gcc-ranlib")
+      FileUtils.ln_s('gcov-7.4', "#{CREW_DEST_PREFIX}/bin/gcov")
+      FileUtils.ln_s('gcov-dump-7.4', "#{CREW_DEST_PREFIX}/bin/gcov-dump")
+      FileUtils.ln_s('gcov-tool-7.4', "#{CREW_DEST_PREFIX}/bin/gcov-tool")
+      FileUtils.ln_s('gfortran-7.4', "#{CREW_DEST_PREFIX}/bin/gfortran")
 
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-c++-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++"
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-g++-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++"
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gcc-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc"
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar"
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm"
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib"
-      system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gfortran-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran"
+      FileUtils.ln_s("$(gcc -dumpmachine)-c++-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++")
+      FileUtils.ln_s("$(gcc -dumpmachine)-g++-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++")
+      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc")
+      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-ar-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar")
+      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-nm-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm")
+      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-ranlib-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib")
+      FileUtils.ln_s("$(gcc -dumpmachine)-gfortran-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran")
```

Comment:
Good catch with the release version.  I'm wondering if we should use a variable to prevent this from happening in the future.